staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,10 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
Having said that, it is totally possible to take those priorities into account. Maybe we should create a separate issue for it and see if we can get it done in 1.8.